pom.xml

```
@@ -77,7 +77,7 @@
     <butterfly.version>1.2.4</butterfly.version>
     <slf4j.version>2.0.7</slf4j.version>
     <log4j.version>2.20.0</log4j.version>
-    <jetty.version>9.4.48.v20220622</jetty.version>
+    <jetty.version>12.0.0.beta0</jetty.version>
```

This seems to be the latest version in the `9.x` series:
```suggestion
    <jetty.version>9.4.51.v20230217</jetty.version>
```